content/webinars/infrastructure-as-code-workshop-2020-04-09/index.md

```diff
@@ -7,6 +7,7 @@ meta_desc: "In this workshop, Luke Hoban shows you the easy way to start using I
 featured: false
 
 aliases: [ "/events/workshop-san-francisco-2020-04-09" ]
+redirect_to: "/webinars/getting-started-with-infrastructure-as-code/"
```

Comment:
Should this be:

```suggestion
redirect_to: "/webinars/getting-started-with-infrastructure-as-code/on-demand/"
```

?

Also, unrelated to this PR, but I noticed it when looking at this, so I'll mention it here... on this page: https://www.pulumi.com/webinars/getting-started-with-infrastructure-as-code/on-demand/

The YouTube video shows the presenters are Luke, Cameron, and Mike, but in the content of the page, it says the Presenters are Paul and Mikhail.

<img width="818" alt="Screen Shot 2020-04-15 at 12 37 08 PM" src="https://user-images.githubusercontent.com/710598/79381001-1fef3580-7f16-11ea-9a50-4c1f10bbb784.png">

Reply:
Fixed the presenters to match the video. I will still redirect them to the "gated" version on the page so we can try to capture the lead.